website/docs/reference/deploy/environment-import-export.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Environment Import & Export
+---
+
+:::info Availability
+
+The environment import and export first appeared in Unleash 4.22.0.
+
+:::
+
+Environment export and import lets you copy feature configurations from one environment to another and even copy features from one Unleash instance to another.
+
+When exporting, you select a set of features and **one** environment to export the configuration from. The environment must be the same for all features.
+
+Then, when you import, you must select **one** environment and **one** project to import into. All features are imported into that project in that environment. If Unleash is unable to import the configuration safely, it will tell you why the import failed and what you need to do fix it (read more about [import rejection](#import-rejection).
+
+## Import & Export items
+
+When you export features, the export will contain both feature-specific configuration and global configuration.
+
+On the project-level these  items are exported:
+
+* [the feature itself](../feature-toggles.mdx)
+* [feature tags](../tags.md)
+
+On the environment-level, these items are exported for the chosen environment:
+
+* [activation strategies](../activation-strategies.md) including [constraints](../strategy-constraints.md) and references to [segments](../segments.mdx)
+* [variants](../feature-toggle-variants.md)
+* enabled/disabled
+
+Additionally, these global configuration items are exported:
+* [custom context fields](../unleash-context.md#context-fields)
+* [feature tag types](../tags.md#tag-types)
+
+Importantly, while references to [segments](../segments.mdx)) are exported, the segments themselves are **not** exported. Consult the [import rejection](#import-rejection) section for more information.
+
+## Export
+
+You can export features either from a project search or the global feature search. Use the search functionality to narrow the results to the list of features you want to export and use the export button to select environment. All features included in your search results will be included in the export.
+
+<Figure caption="Search-driven export screen with WYSIWYG mode."  img="/img/export.png"/>
+
+## Import
+
+Import is a 3 stage process designed to be efficient and error-resistant.
+
+### Import stages
+
+* **upload** - you can upload previously exported JSON file or copy-paste export data from the exported JSON file into the code editor
+* **validation** - you will get feedback on any errors or warnings before you do the actual import. This makes sure your feature flags configurations. You will not be able to finish the import if you have errors. Warnings don't stop you from importing.
+* **import** - the actual import that creates a new configuration in the target environment or creates a [change request](../change-requests.md) when the environment has change requests enabled
+
+![Three-staged import process](/img/import.png)
+
+### Import rejection
```

Comment:
Maybe we can rephrase this header and section to be "import requirements" instead of import rejection?

```suggestion
### Import requirements
```